pkg/master/master.go

```diff
@@ -217,12 +190,16 @@ func New(c *Config) (*Master, error) {
 	if c.RESTStorageProviders == nil {
```

Comment:
Can we get rid of `c.RESTStorageProviders` and turn it into a parameter of `InstallAPIs`? I think it's not used anywhere else.